db/migrate/20250425102555_validates_add_ons_taxes_organizations_foreign_key.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ValidatesAddOnsTaxesOrganizationsForeignKey < ActiveRecord::Migration[7.2]
+  def change
+    validate_foreign_key :add_ons_taxes, :organizations
```

Comment:
Can we enforce the validation before the data migration (the job) is over 🤔

Reply:
this is only needed because adding a Foreign key with `validate:false` is quicker and requires less locks. Once the foreign key is validated it will be impossible to insert invalid data. (strong-migrations suggests to split up these migrations). So no data migration needs to take place before validating the foreign key afaik.

Reply:
I realize I was confusing validating that it's a foreign key and validating that it's not nullable 🤦‍♂️ 
Thank you!